### PR TITLE
add cneuromonth videos post

### PR DESCRIPTION
**Pull request by FrancoisPgm**

### CODE REVIEW

File: content/posts/cneuromonth_videos.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: "CNeuromonth videos"
+description: "The videos of our presentations for the CNeuromont 2021 are available on our youtube channel."
+date: 2021-04-21 
+author: "François Paugam"
+image: "cneuromonth_videos/youtube-logo.png"
+draft: false
+tags: ["cneuromonth"]
+---
+
+Hello, I am François Paugam, a PhD student and the one in chatrge of the special effects for the great videos we made for the CNeuromod month, our online progress report events. If you missed it, or if you are just curious to see what we are doing in the numerous and diverse branches of the Neuromod project, or if you enjoyed the presentations so much that you want to watch them over and over, you can access them on our [youtube channel](https://www.youtube.com/channel/UCjeuWWEmZt5Zbx0LBiPbDJw).
```
Comment (vborghe) [inline content/posts/cneuromonth_videos.md]: typo: chatrge > charge

Comment (vborghe) [inline content/posts/cneuromonth_videos.md]: minor comment: perhaps better to stick with CNeuromod everywhere?

**Comment by vborghe** [comment]
@pbellec would you like to have [this table](https://github.com/courtois-neuromod/cneuromod.ca/blob/master/content/posts/cneuromod_month_2021.md) at the end of the post with direct link to specific talks?

File: content/posts/cneuromonth_videos.md (hunk 1 above)
Comment (FrancoisPgm) [reply content/posts/cneuromonth_videos.md]: Yes that would be better, I also changed CNeuromonth for CNeuromod month to be consistant with Julie's previous post.

Comment (FrancoisPgm) [reply content/posts/cneuromonth_videos.md]: thanks for cathcing that

**Comment by lunebellec** [comment]
So I would personally be in favour of adding the table. It will be a nice reference moving forward to check the content of the cneuromod month. Alternatively we could do a playlist on youtube, not 100% sure of the added value of the table - besides that it's nice and compact.

Also maybe we could include a picture of the recording set up.

**Comment by vborghe** [comment]
Oh, plus one for the recording set up photo! People will be curious!

File: content/posts/cneuromonth_videos.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "CNeuromonth videos"
```
Comment (vborghe) [inline content/posts/cneuromonth_videos.md]: I'll stop being annoying but I wonder if the title can be something like CNeuromonth 2021 - the videos!
I'm just thinking of how it is going to look on the blog main page..
Screen Shot 2021-04-22 at 8 20 22 AM

Comment (lunebellec) [reply content/posts/cneuromonth_videos.md]: provide feedback = awesome :hugs: :hugs: :hugs:  $/neq$ annoying 

I agree with proposed title.

**Comment by lunebellec** [comment]
The preview seems broken, but locally it looks fine. I am merging, and will fix if it turns out the deployment on netlify is indeed broken. But there are lots of glitches with the hugo preview, so I am hopeful.

**Pull request merged**